src/Compilers/VisualBasic/Portable/Symbols/SynthesizedSymbols/SynthesizedParameterSymbol.vb

```diff
@@ -233,6 +233,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return _name
             End Get
         End Property
+
+        Friend NotOverridable Overrides Sub AddSynthesizedAttributes(compilationState As ModuleCompilationState, ByRef attributes As ArrayBuilder(Of SynthesizedAttributeData))
+            Dim compilation = Me.DeclaringCompilation
+            If Type.ContainsTupleNames() AndAlso
+                compilation.HasTupleNamesAttributes AndAlso
```

Comment:
More notes from Friday:

I thought missing string was already being tested, but looking at it today, it just looks like the general case of "missing string produces diagnostic" is being tested: AttributeTests_Tuples.vb -> `TupleInDeclarationFailureWithoutString` (which tests that a tuple return type with names produces a diagnostic, does not test parameters of property setters)

However, `TupleElementNamesAttribute` missing is explicitly tested: AttributeTests_Tuples.vb -> `TupleAttributeMissing`. This uses the `s_tuplesTestSource` string, which is a test resource that contains many different positions of tuples (all possible positions?), including our case of a setter.

I'm a little suspect that we're not testing a lack of property names, I'm guessing I just haven't found the test yet. However, I do still need to add the test for String missing, I think I'll convert the existing test (that just tests method return types) to use the same "use everything" test resource, without String.